src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
-body {
-  margin: 0;
+@import "./variables";
+
+.page {
+  font-family: $font;
+  font-size: 30px;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  width: $cell-size * 7 + $gap-size * 6 + 20px;
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.